contrib/gitian-build.py

```diff
@@ -196,13 +197,21 @@ def main():
         exit(1)
 
     # Add leading 'v' for tags
+    if args.commit and args.pull:
+        raise Exception('Cannot have both commit and pull')
     args.commit = ('' if args.commit else 'v') + args.version
-    print(args.commit)
 
     if args.setup:
         setup()
 
     os.chdir('bitcoin')
+    if args.pull:
+        subprocess.check_call(['git', 'fetch', args.url, 'refs/pull/'+args.version+'/merge'])
+        os.chdir('../gitian-builder/inputs/bitcoin')
+        subprocess.check_call(['git', 'fetch', args.url, 'refs/pull/'+args.version+'/merge'])
+        args.commit = subprocess.check_output(['git', 'show', '-s', '--format=%H', 'FETCH_HEAD'], universal_newlines=True).strip()
+        args.version = 'pull-' + args.version
+    print(args.commit)
```

Comment:
is this intended or a left-over debug statement?

Reply:
That's intended, move from the above one.